src/View/Helper/TimeHelper.php

```diff
@@ -33,6 +33,31 @@ class TimeHelper extends Helper
     use StringTemplateTrait;
 
     /**
+     * Config options
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'outputTimezone' => null
+    ];
+
+    /**
+     * Get a timezone.
+     *
+     * Will use the provided timezone, or default output timezone if defined.
+     *
+     * @param null|string|DateTimeZone $timezone The override timezone if applicable.
+     * @return null|string|DateTimeZone The chosen timezone or null.
+     */
+    protected function _timezone($timezone)
```

Comment:
I wonder if this should read _getTimezone() to be clear this is more a getter than a setter - in light of future set/get cleanups this could be useful to already do with new methods.